rollup.config.mjs

```diff
@@ -45,23 +47,22 @@ if (process.env.BUILD === 'dist') {
     },
     external: (id) => !id.startsWith('.') && !path.isAbsolute(id),
     plugins: [
-      nodeResolve({browser: true}),
       babel({
         babelHelpers: 'bundled',
```

Comment:
btw: I hope no helpers are injected by babel? 🤔  Should we maybe use `external` to ensure things crash if they accidentally are added? https://github.com/rollup/plugins/tree/master/packages/babel#babelhelpers